Formula/llvm@7.rb

```diff
@@ -29,6 +29,10 @@ class LlvmAT7 < Formula
   resource "clang" do
     url "https://releases.llvm.org/7.1.0/cfe-7.1.0.src.tar.xz"
     sha256 "e97dc472aae52197a4d5e0185eb8f9e04d7575d2dc2b12194ddc768e0f8a846d"
+    patch do
+      url "https://gist.githubusercontent.com/hjelmn/9fa1a7742acad71f404544a03bda414a/raw/b56a25f7208023d9724f671dfd7a80cbdac2a612/cfe-7.1.0-darwin-arm64.patch"
+      sha256 "eb6be925cdab150f5d01d51fa6bd6a4fe1dbf481794b83976ff877524192d1c1"
+    end
```

Comment:
These are gonna need to be submitted upstream to be considered for inclusion in homebrew

Reply:
[LLVM 7 isn't updated anymore upstream](https://github.com/llvm/llvm-project/commits/release/7.x), this is basically a legacy package.

Reply:
If it's abandoned it should be deprecated and we still shouldn't use random patches that aren't submitted upstream

Reply:
These changes are all upstream. Just bavkported to llvm7 because it is no longer maintained. The only reason I request this is llvm7 is needed to build the ghc llvm backend. There is no native backend for ARM64 yet.

Reply:
I suppose the alternative is I make a keg, correct?